frontend/src/Components/Nav.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Homepage from './pages/Homepage';
+
+function Nav() {
```

Comment:
Function name could be improved as it is quite ambiguous

Reply:
navigation

Reply:
How will this link to the homepage?

Reply:
it will link to every page